app/javascript/article-form/components/EditorActions.jsx

```diff
@@ -96,6 +96,7 @@ export const EditorActions = ({
           onClick={onClearChanges}
           className="whitespace-nowrap fw-normal"
           size="s"
+          aria-label="Revert new changes"
```

Comment:
I added this to be able to retrieve the button by label in the E2E tests. Looks like cypress-testing-library or dom-testing-library isn't able to deduce that the button text is `Revert new changes`

Reply:
refactor (blocking) - I'm not super-keen to add an `aria-label` for testing purposes, when we already have exposed button text 🤔  

Locally I deleted this aria label though and the tests passed OK.
`findByText(/^Revert new changes$/i)` doesn't work because of the `span` but the way you have targeted it with `findByRole('button', { name: /^Revert new changes$/i })` seems to work fine without the additional label.

If you can double check it works from your side too it would be good to remove the additional `aria-label`

Reply:
Weird, I had tried that yesterday. I'll revert the aria-label and try that again. Thanks!

Reply:
Done